### DISCUSSION TITLE
How to build an executable JAR server package with dependencies

### DISCUSSION THREAD

**Discussion by Alima777** [apache/iotdb#3713] [Q&A] [answered]
如何创建 server 可执行的JAR包？

How to build an executable JAR server package with dependencies

**Comment by HTHou** [comment, marked as answer]
Add `-P get-jar-with-dependencies`.

Also, make sure there is a profile likes below in the pom file.
https://github.com/apache/iotdb/blob/394467bd701331c58742eb84e684e80ea85688c8/session/pom.xml#L125-L151
